apps/metadata-converter/src/app/components/record-form/record-form.component.ts

```diff
@@ -100,6 +98,9 @@ export class RecordFormComponent implements AfterViewInit {
       keywords: hasPrevious ? this.record.keywords : [],
       topics: hasPrevious ? this.record.topics : [],
       onlineResources: [],
+      otherLanguages: [],
+      defaultLanguage: 'en',
+      overviews: [],
```

Comment:
I have adjusted these fixtures to make them correct; there were typing errors that were hidden because of the cast to CatalogRecord below